modules/abstract-eth/package.json

```diff
@@ -40,7 +40,7 @@
     "@bitgo/sdk-coin-eth": "^1.1.0",
     "@bitgo/sdk-core": "^1.1.0",
     "@bitgo/statics": "^7.0.0",
-    "bignumber.js": "^8.0.1",
-    "bip32": "^2.0.6"
+    "@bitgo/utxo-lib": "^2.4.1",
```

Comment:
Can we upgrade `bip32` and use that instead of using `utxo-lib`?

Reply:
Sadly no. BIP32 (and ECPair) can no longer be used standalone, because they require initialization with a chosen secp256k1 library. In utxo-lib, we have the code to wrap noble-secp256k1 appropriately and initialize BIP32 / ECPair, and export them for use elsewhere in the SDK.

Reply:
We should deprecate adding features to `utxo-lib` and add new features to other packages like `abstract-utxo` and `sdk-core` since that (and coin specific functionality) will be migrating into.

Reply:
What's the difference between `utxo-lib` and `abstract-utxo`?

Reply:
Ok, that makes sense. Doesn't seem like there are implications for this PR (beyond moving the ecc stuff to core), since the code I'm importing from bitcoinjs-lib is needed by code still in `utxo-lib`.

I'll have to think a bit more about the implications of this for the PSBTification work that I'm starting on.

Possibly it means that for Doge, ZCash, Litecoin, Bitcoin, BTG, BCH and BSV any extensions needed to support each coin in `psbt` will only be made in their specific modules and no PSBT wrapper will be added to `abstract-utxo` nor `utxo-lib`.

Reply:
> abstract-utxo is the shared utility between all the utxo coins

On wallet platform as well? The `utxo-lib` package is a pretty low-level wrapper around `bitcoinjs-lib` that adds some additional methods for other networks.

> Basically account-lib and utxo-lib will be completely deprecated as entry points into the sdk ecosystem.

The `utxo-lib` package was never meant as an entrypoint to the sdk ecosystem, but as a low-level systems library that can be used across WP/IMS/SDK etc

Reply:
`account-lib` has been stripped to the bones. I don't think `utxo-lib` will go away for quite awhile since it's a low priority item. `abstract-utxo` and `abstract-eth` have taken over the roles of `account-lib` and `utxo-lib` so while it may not be completely deprecated, it may become the same shell that `account-lib` is now.

Reply:
We also have been maintaining backwards compatibility for now, so when that does happen, it would probably end up being a shell that exports what it can without causing circular references. Something along those lines, but not on the critical development path currently.

Reply:
I'm not sure how `abstract-utxo` has taken over the role of `utxo-lib`. From what I can tell the roles have remained steady over the last couple of years.

Reply:
`abstract-utxo` is brand new along with `abstract-eth` due to the modularization efforts. The only reason currently more of `utxo-lib` wasn't ported over was due to time constraints.

Reply:
The `abstract-utxo` package may be new, but the `AbstractUtxo` class has existed for years. It exists to implement the `coin` interface defined in BitGoJS. 

The `utxo-lib` library serves a different purpose: it extends `bitcoinjs-lib` (with minimal dependencies) to support more values for the `network` parameter.